service/src/main/kotlin/fi/espoo/evaka/shared/security/AccessControl.kt

```diff
@@ -363,6 +362,7 @@ WHERE employee_id = :userId
             createUnits = hasPermissionFor(user, Action.Global.CREATE_UNIT),
             vasuTemplates = user.hasOneOfRoles(UserRole.ADMIN),
             personalMobileDevice = user.hasOneOfRoles(UserRole.UNIT_SUPERVISOR),
+            pinCode = !user.hasOneOfRoles(UserRole.FINANCE_ADMIN),
```

Comment:
A negative check seems a bit strange to me :thinking: 
Why do we explicitly take out FINANCE_ADMIN, but everybody else (including users with no roles at all) have permission?

Reply:
The idea was to communicate intent: Everyone else except FINANCE_ADMIN. Fixed by listing all roles and adding a comment instead.

Reply:
Yeah I understood the intent (except for completely role-less users), but it seems like a strange requirement. For example, why does DIRECTOR need access to pin code functionality?